forbidigo/forbidigo.go

```diff
@@ -24,10 +24,15 @@ type UsedIssue struct {
 	identifier string
 	pattern    string
 	position   token.Position
+	customMsg  string
 }
 
 func (a UsedIssue) Details() string {
-	return fmt.Sprintf("use of `%s` forbidden by pattern `%s`", a.identifier, a.pattern)
+	explanation := fmt.Sprintf(": `%s`", a.customMsg)
```

Comment:
I think regular quotes rather than backticks might be more appropriate here because we are dealing with a string rather than an identifier:

```suggestion
	explanation := fmt.Sprintf(`because "%s"`, a.customMsg)
```